pennylane/transforms/optimization/single_qubit_fusion.py

```diff
@@ -14,19 +14,238 @@
 """Transform for fusing sequences of single-qubit gates."""
 # pylint: disable=too-many-branches
 
+from functools import lru_cache
+from typing import Optional
+
 import pennylane as qml
 from pennylane.ops.qubit import Rot
 from pennylane.queuing import QueuingManager
 from pennylane.tape import QuantumScript, QuantumScriptBatch
 from pennylane.transforms import transform
-from pennylane.typing import PostprocessingFn
+from pennylane.typing import PostprocessingFn, TensorLike
 
 from .optimization_utils import find_next_gate, fuse_rot_angles
 
 
+@lru_cache
+def _get_plxpr_single_qubit_fusion():  # pylint: disable=missing-function-docstring,too-many-statements
+    try:
+        # pylint: disable=import-outside-toplevel
+        from jax import make_jaxpr
+
+        from pennylane.capture import PlxprInterpreter
+        from pennylane.operation import Operator
+    except ImportError:  # pragma: no cover
+        return None, None
+
+    # pylint: disable=redefined-outer-name
+
+    class SingleQubitFusionInterpreter(PlxprInterpreter):
+        """Plxpr Interpreter for applying the ``single_qubit_fusion`` transform to callables or jaxpr
+        when program capture is enabled.
+
+        .. note::
+
+            In the process of transforming plxpr, this interpreter may reorder operations that do
+            not share any wires. This will not impact the correctness of the circuit.
+        """
+
+        def __init__(self, atol: Optional[float] = 1e-8, exclude_gates: Optional[list[str]] = None):
```

Comment:
Why not default to an empty list? That way we don't need to worry about handling `exclude_gates = None`?
```suggestion
        def __init__(self, atol: Optional[float] = 1e-8, exclude_gates: Optional[Sequence[str]] = ()):
```

Reply:
Good point. Although the default value is different in the legacy transform, this shouldn't change anything so I am going to do it